R/pkg.R

```diff
@@ -283,66 +283,99 @@ getPackageRecords <- function(pkgNames,
   allRecords
 }
 
+# Return TRUE when the data frame for this package has the given RemoteType.
+hasRemoteType <- function(df, remoteType) {
+  # Do not compare with 'identical'; RemoteType may be a factor.
```

Comment:
Agh. I'd prefer to know where this is happening so that we can fix the problem at the source, but this should still be okay.

Reply:
Would you rather I use the null/equality checks inline at each call-point? (reverting most of your https://github.com/rstudio/packrat/commit/29a4fdcb5d34113222b73e4eb2affd0e3fc3f1fd commit)

Reply:
No, I think this is fine. Thanks!